### PR TITLE
refactor: make next4 compliant

### PR DESCRIPTION
**Pull request by Lemii**
This PR changes the following:

- ~~Update dependencies to use the latest `next4` version~~
- Changes random business/bridgename `name` generation to comply with new validation (it previously used `Math.random()` which returns an illegal period character)
- Changes default uri values to comply with new validation

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: package.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
         "dev": "nodemon ./index.js"
     },
     "dependencies": {
-        "@arkecosystem/crypto": "^2.6.0-next.1",
-        "@arkecosystem/core-utils": "^2.6.0-next.1",
-        "@arkecosystem/core-magistrate-crypto": "^2.6.0-next.1"
+        "@arkecosystem/crypto": "^2.6.0-next.4",
```
Comment (faustbrian) [inline package.json]: Those bumps aren't necessary as `^` indicates that it should pick any release that matches `^2.6.0-next.X` where `X` has to be higher than the previous value. You can simply run `yarn install` or `yarn upgrade` to update to the latest version as long as it is in the `^2.6.0-next.X` range.

Comment (Lemii) [reply package.json]: Gotcha. I've reverted the commit :+1:

**Review by spkjp — APPROVED** [review]

**Comment by ghost** [comment]
A collaborator has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged but was not assigned a bounty tier. A maintainer will assign a bounty tier to this pull request in the next few days.

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

**Pull request merged**